lib/chef/cookbook/metadata.rb

```diff
@@ -56,9 +55,8 @@ class Metadata
 
       COMPARISON_FIELDS = [ :name, :description, :long_description, :maintainer,
                             :maintainer_email, :license, :platforms, :dependencies,
-                            :providing, :attributes, :recipes, :version,
-                            :source_url, :issues_url, :privacy, :chef_versions, :ohai_versions,
-                            :gems ].freeze
+                            :providing, :recipes, :version, :source_url, :issues_url,
+                            :privacy, :chef_versions, :ohai_versions, :gems ].freeze
```

Comment:
i spent way too long diffing these lines in my head.

Reply:
that's why 1 item per line is a totally inefficient, but great way to do arrays